.github/workflows/test-examples.yaml

```diff
@@ -7,11 +7,11 @@ jobs:
 
   test-examples:
     name: Test Examples
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
 
     env:
       SHELL: /bin/bash
-      INSTALLED_EDM_VERSION: "3.4.0"
+      INSTALLED_EDM_VERSION: "3.7.0"
```

Comment:
why not 4 ?

Reply:
Because 3.7 was what is on my local computer :p. Updating to 4 here.